alws/build_planner.py

```diff
@@ -135,16 +140,62 @@ async def init_build_repos(self):
     async def add_linked_builds(self, linked_build):
         self._build.linked_builds.append(linked_build)
 
-    def remove_force_to_build_artifacts_from_template(
-        self,
-        module: ModuleWrapper,
-        build_task_refs: typing.List[build_schema.BuildTaskRef],
-    ):
-        # if some refs with added artifacts in module template
-        # was enabled to build, we should delete them from template
-        for ref in build_task_refs:
+    @staticmethod
+    async def get_multilib_artifacts(
+            task: build_schema.BuildTaskModuleRef, has_devel: bool = False
+    ) -> typing.List[dict]:
+        beholder_client = BeholderClient(
+            settings.beholder_host, token=settings.beholder_token)
+        major_distr_version = task.module_platform_version.split('.')[0]
+        multilib_artifacts = []
+
+        # FIXME: Hardcoded platform for now, should send actual platform
+        #  in the task ref
+        multilib_packages = await MultilibProcessor.get_module_multilib_data(
+            beholder_client, 'AlmaLinux', major_distr_version,
```

Comment:
We can use platform data from `self._platforms`, i'm think there also should be `distr_version`

Reply:
Makes sense, will fix the code